src/WorkflowManager/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -365,15 +358,29 @@ private async Task HandleDicomExportAsync(WorkflowRevision workflow, WorkflowIns
                         workflowInstance.BucketId,
                         artifact,
                         true);
-                    if (objects.IsNullOrEmpty() is false)
+
+                    var dcmFiles = objects?.Where(o => o.FilePath.EndsWith(".dcm"))?.ToList();
+
+                    if (dcmFiles?.IsNullOrEmpty() is false)
                     {
-                        files.AddRange(objects.ToList());
+                        files.AddRange(dcmFiles.ToList());
                     }
                 }
             }
 
             artifactValues = files.Select(f => f.FilePath).ToArray();
 
+            if (artifactValues.IsNullOrEmpty())
+            {
+                _logger.ExportFilesNotFound(task.TaskId, workflowInstance.Id);
```

Comment:
This should probably be at least at Warning Log Level, maybe even Error !

Reply:
Changed this to be Error.